### PR TITLE
App styles

### PR DESCRIPTION
**Pull request by colejoh**

### CODE REVIEW

File: src/assets/_vars.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-$font-body: 'Avenir Next', sans-serif;
```
Comment (nickysemenza) [inline src/assets/_vars.scss:1]: if we're done with Avenir Next, you can go ahead and kill it from `src/fonts/`

File: src/index.js [hunk 2]
```diff
@@ -31,6 +31,6 @@ ReactDOM.render(
     <Provider store={store}>
         <App />
     </Provider>,
-  document.getElementById('root')
```
Comment (nickysemenza) [inline src/index.js:34]: why this?

File: src/assets/index.scss [hunk 3]
```diff
@@ -69,3 +71,8 @@ p {
 .icon {
   fill: red;
 }
+.max-width {
```
Comment (nickysemenza) [inline src/assets/index.scss]: imo it's kind of confusing to have a custom class name that's the same as a standard property name- perhaps name it something that describes it's purpose, `fullWidthContainer` or something along those lines maybe?

File: src/pages/Application/ApplicationForm.js [hunk 4]
```diff
@@ -26,65 +27,99 @@ class ApplicationForm extends Component {
...
+                                  <button onClick={this.toggleItem.bind(this,'has_no_github')} className="opt-out-button">No thanks</button>
+                              }
+                          </div>
+                      }
+                    </div>
+                </div>
+                <div className="app-row">
+                    <div className="col-6">
+                        <label>Gender</label>
+                        {/* <Select ref="schoolSelect"
```
Comment (nickysemenza) [inline src/pages/Application/ApplicationForm.js]: i know it's commented out right now, but this should end up in it's own component eventually- even though it isn't strictly a reusable component, it's cleaner and keeps redux logic outside of `<ApplicationForm/>`

**Review by nickysemenza — CHANGES_REQUESTED** [review]
Looks 🔥  overall, just a few misc comments / thoughts.

Might be good to fix the UI bug on the interest signup box so that this can get merged in to master

File: public/index.html [hunk 5]
```diff
@@ -4,34 +4,12 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
-    <!--
-      manifest.json provides metadata used when your web app is added to the
-      homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
-    -->
+    <link href="https://fonts.googleapis.com/css?family=Fira+Sans:400,500,700" rel="stylesheet">
```
Comment (kedarv) [inline public/index.html:7]: Are we using all of these weights?

File: src/assets/_application.scss [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+  margin-top: 8px;
+
+  input:active {
+    border: none;
+  }
+}
+.paddingx {
+  padding: 0 16px;
+}
+.paddingr {
```
Comment (kedarv) [inline src/assets/_application.scss]: nitpick: I don't like these names for padding and margin modifiers

File: src/assets/_application.scss [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+  font-family: $font-body;
+  margin-top: 8px;
+  font-size: 14px;
+  outline: none;
+
+  &:active {
+    border: 2px solid $darkBlue;
+  }
+}
+.Select-control {
```
Comment (kedarv) [inline src/assets/_application.scss]: lowercase `s`

**Review by kedarv — CHANGES_REQUESTED** [review]
Mostly nitpicky fixes on my part, take a look at review from @nickysemenza and feel free to merge

File: src/assets/_application.scss (hunk 7 above)
Comment (colejoh) [reply src/assets/_application.scss]: This is a component that we import (react select) I don't have control over the class names, just trying to overwrite the styling.

File: src/assets/_application.scss (hunk 6 above)
Comment (colejoh) [reply src/assets/_application.scss]: We need to define a standard. What do you prefer?

File: public/index.html (hunk 5 above)
Comment (colejoh) [reply public/index.html:7]: Likely not. I was going to take out either the 500 or the 700 when I figured out which one looked better.

File: src/pages/Application/ApplicationForm.js (hunk 4 above)
Comment (colejoh) [reply src/pages/Application/ApplicationForm.js]: There's a lot of refactoring that needs to be done.

File: src/assets/index.scss (hunk 3 above)
Comment (colejoh) [reply src/assets/index.scss]: I don't think it's bad. I've seen it done elsewhere.

File: src/assets/_vars.scss (hunk 1 above)
Comment (colejoh) [reply src/assets/_vars.scss:1]: Will do

File: src/assets/_application.scss (hunk 7 above)
Comment (nickysemenza) [reply src/assets/_application.scss]: fyi - https://github.com/JedWatson/react-select#custom-classnames

File: src/assets/_application.scss (hunk 6 above)
Comment (nickysemenza) [reply src/assets/_application.scss]: we should have a discussion on slack probably, IMO some variation of [BEM](https://css-tricks.com/bem-101/) usually works well, but open to other suggestions

File: src/assets/_application.scss (hunk 7 above)
Comment (nickysemenza) [reply src/assets/_application.scss]: @colejoh apart from this, everything look good i think

**Review by nickysemenza — APPROVED** [review]

**Comment by nickysemenza** [comment]
@colejoh Merging this in, it'd be good to clean up the styling of the Select component like so: https://github.com/BoilerMake/frontend/pull/24#discussion_r129917833

Otherwise, great first PR 😄

**Pull request merged**